src/_guides/language/language-tour.md

```diff
@@ -452,30 +452,30 @@ const msUntilRetry = secondsUntilRetry * msPerSecond;
 A Dart string is a sequence of UTF-16 code units. You can use either
 single or double quotes to create a string:
 
-<!-- language-tour/quoting/bin/main.dart -->
+<?code-excerpt "language-tour/quoting/bin/main.dart"?>
 {% prettify dart %}
-var s1 = 'Single quotes work well for string literals.';
-var s2 = "Double quotes work just as well.";
-var s3 = 'It\'s easy to escape the string delimiter.';
-var s4 = "It's even easier to use the other delimiter.";
+String s1 = 'Single quotes work well for string literals.';
```

Comment:
...and here...

Reply:
These variable declarations appear in the section on "Strings", which may be why the variables are declared as `String`. (The same is true of the use of `int` and `double` above, in the sections on "Numbers".)

Reply:
But — the original page doesn't have the type annotations, right? https://www.dartlang.org/guides/language/language-tour#strings

Let's keep it that way.

Reply:
> Let's keep it that way.

Ok, I'll revert those code changes that were done before the code was copied over into this repo